src/main/java/org/ibp/api/java/impl/middleware/dataset/DatasetExcelExportServiceImpl.java

```diff
@@ -67,17 +67,7 @@ public List<MeasurementVariable> getColumns(final int studyId, final DatasetDTO
 		} else {
 			columns = this.studyDatasetService.getSubObservationSetVariables(studyId, dataSet.getDatasetId());
 		}
-
-		if (includeSampleGenotypeValues) {
-			// Add Genotype Marker variables to the list of columns
-			final SampleGenotypeVariablesSearchFilter filter = new SampleGenotypeVariablesSearchFilter();
-			filter.setStudyId(studyId);
-			filter.setDatasetIds(Arrays.asList(dataSet.getDatasetId()));
-			final Map<Integer, MeasurementVariable> genotypeVariablesMap =
-				this.sampleGenotypeService.getSampleGenotypeVariables(filter);
-			columns.addAll(genotypeVariablesMap.values());
-		}
-
+		this.includeSampleGenotypeValues(studyId, dataSet, includeSampleGenotypeValues, columns);
```

Comment:
Rename includeSampleGenotypeValues to isIncludeSampleGenotupeValues, so that it's not the same with the method name.